c/extras/py_ftrl.cc

```diff
@@ -334,6 +339,33 @@ void Ftrl::fit(const PKArgs& args) {
     default:             throw TypeError() << "Cannot predict for a column "
                                            << "of type `" << stype_y << "`";
   }
+
+  if (feature_names == nullptr) {
+    size_t nfeatures = (*dtft)[0]->get_nfeatures();
+    size_t ncols = dt_X->ncols;
+    std::vector<std::string> column_names = dt_X->get_names();
```

Comment:
```suggestion
    const std::vector<std::string>& column_names = dt_X->get_names();
```
to avoid unnecessary copy

Reply:
Yep, also missed.